copter/source/docs/traditional-helicopter-tips.rst

```diff
@@ -0,0 +1,33 @@
+.. _traditional-helicopter-tips:
+
+===========================
+Traditional Helicopter Tips
+===========================
+
+- Be sure to follow the wiki instructions, step by step!
+- Be sure the CG is in line with the main rotor shaft
+- Blade tracking must be within a quarter blade thickness or better to avoid excessive vibration
+- Vibration is the arch enemy of Helicopter autopilots!
+   - Internal Combustion Engines and Turbines can created high frequency vibrations that do not appear in the VIBE logs and can be aliased to low frequencies which can impact EKF and Attitude control (see below for ground tests to check for this). This also often occurs in electric helicopters.
+   - The autopilot should be well isolated for vibration. Using autopilots with internal IMU vibration damping and vibration reducing mounts is recommended.
+- Traditional Helicopter *is* capable of inverted flight and using full negative collective ranges.
+
+Checking for High Frequency Vibration
+=====================================
+
+Symptoms of high frequency vibration can be that the attitude estimate becomes incorrect causing it to lean significantly even though the pilot is commanding level flight in stabilized modes. In the extreme, it can result in an :ref:`ekf-inav-failsafe` or even a crash. Excessive vibration can also make altitude hold imprecise or even produce uncontrollable climbs/descents.
+
+One method of testing this, is to secure the helicopter to the ground, and run up the main rotor, watching the artificial horizon in the ground control station. If it moves away from level, you probably have a vibration issue which needs to be addressed. Testing should be done at various collective levels and rotor speeds. Be careful not to over-stress the helicopter with collective extremes.
+
+Setup for ground test:
+
+#. Secure vehicle to the ground.
+#. Temporarily reduce Roll, Pitch, and Yaw  P,I,D,and FF PIDs to zero. Be sure to restore them after this test!
```

Comment:
If you zero the FF gain then you cant control the swashplate.  I think zero PIDs is best and leave FF to retain control

Reply:
done